kf_lib_data_ingest/app/cli.py

```diff
@@ -234,12 +264,14 @@ def test(
     file or a path to a directory which contains a file called
     `ingest_package_config_path.py`
     """
-
     # Make kwargs from options
     frame = inspect.currentframe()
     args, _, _, values = inspect.getargvalues(frame)
     kwargs = {arg: values[arg] for arg in args[1:]}
 
+    if kwargs.pop("no_validate"):
+        kwargs["validation_mode"] = None
+
```

Comment:
Not needed, because arguments are passed to def ingest which also does this.

Reply:
Not sure what you mean by this? You mean the ingest pipeline? CLI.ingest has two different options `no_validate` and `validation_mode`, and then the ingest pipeline only gets the `validation_mode` var.